runtime/chainerx_util.h

```diff
@@ -43,5 +43,14 @@ bool IsFloat(chainerx::Dtype dtype);
 
 void BlitArray(const chainerx::Array& src, const chainerx::Array& dst);
 
+chainerx::Array MatMul(const chainerx::Array& a, const chainerx::Array& b);
+chainerx::Array Conv(
```

Comment:
Again, it'd be safe to intentionally avoid the same name with ChainerX's. `Convolution`?